app/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Amar Muha</title>
+  <link rel="stylesheet" href="styles/style.css">
+</head>
+
+<body>
+  <div class="wrapper">
+    <section class="header-section">
+      <header class="main-header">
+        <div class="logo">
+          Amar
+        </div>
```

Comment:
- [ ] Please I highly recommend wrapping the `logo` with an anchor `a` tag with links. This helps improves user experiences, provides HTML semantics and conventions. Kindly [check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more information. 😄